src/css/base.scss

```diff
@@ -316,14 +316,14 @@ div {
 }
 
 .correct-answer__display {
-  height: 300px;
-  width: 300px;
-  filter: invert(1);
+  height: 400px;
```

Comment:
This seems like a great place to use SCSS "extends" since you have repeated CSS for the displays sharing that CSS.